File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hashmap=defaultdict(int)
        comb=0
        for i in range(len(nums)):
            comb=target-nums[i]
            if comb in hashmap:
                return [hashmap[comb],i]
            hashmap[nums[i]]=i
            

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> map=new HashMap<Integer,Integer>();
        int comb=0;
        for(int i=0;i<nums.length;i++){
            comb=target-nums[i];
            if(map.containsKey(comb)){
                return new int[]{map.get(comb),i};
            }
            map.put(nums[i],i);
        }
        return new int[]{,};

    }
}